netbox/dcim/tests/test_filtersets.py

```diff
@@ -3116,6 +3144,15 @@ def test_device(self):
         params = {'device': [devices[0].name, devices[1].name]}
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
 
+    def test_virtual_chassis_member(self):
+        # Device 1 & 3 have 1 management interface, Device "None" has 5 interfaces, 2 of which are management and
+        # will not be visible
```

Comment:
The fourth device exists solely to serve as the opposite end of the various cable connections. It would probably be simpler to avoid using it as a VC member, and instead create a separate device. Then we could have e.g. Device 1A and Device 1B, so it's more readily apparent that a VC is in use. (We also don't want to reference "Device None:" If it needs to be referenced, we should give the device a proper name.)

Reply:
I am going to rename "Device 1" to "Device 1A" and then add in "Device 1B" to the mix with a new interface.  I am going to try and exclude 1B from as many of the other tests as possible (by using a non-tested role/device_type/etc)